vivisect/qt/funcgraph.py

```diff
@@ -277,6 +300,31 @@ def _hotkey_deczoom(self):
         #self.vw.vprint("NEW ZOOM    %f" % newzoom)
         self.mem_canvas.setZoomFactor(newzoom)
 
+    @pyqtSlot()
```

Comment:
any python "callable" is actually a PyQt slot, so the pyqtSlot is only really useful when you want to declare what types must be handed in ( and in pyqt5 I think they're doing away with the decorator as a whole ).  No harm in having it, but in it's current form is un-needed